sdk_container/src/third_party/coreos-overlay/sys-kernel/coreos-sources/coreos-sources-6.6.48.ebuild

```diff
@@ -38,4 +38,8 @@ UNIPATCH_LIST="
 	${PATCH_DIR}/z0001-kbuild-derive-relative-path-for-srctree-from-CURDIR.patch \
 	${PATCH_DIR}/z0002-revert-pahole-flags.patch \
 	${PATCH_DIR}/z0003-Revert-x86-boot-Remove-the-bugger-off-message.patch \
+	${PATCH_DIR}/z0004-efi-Add-an-EFI_SECURE_BOOT-flag-to-indicate-secure-b.patch \
```

Comment:
If you can add please the initial source of the patches in the commit message too, so that we can have a better tracking of those (https://sources.debian.org/src/linux/6.1.94-1/debian/patches/features/all/lockdown/). Question: are there any newer sources for the 6.6 and 6.10 that we can leverage, or maybe an lkml thread?

Reply:
> I'll update the commit message. No newer lkml submissions, there is a 6.6 branch from debian https://sources.debian.org/src/linux/6.6.13-1~bpo12%2B1/debian/patches/features/all/lockdown/ but i think i took the 6.1.94 version, manually applied to 6.6.47 and then exported them.

It's great we can rely on a newer version and hopefully things won't break that often :).